components/movementsensor/movementsensor.go

```diff
@@ -112,17 +122,31 @@ func Named(name string) resource.Name {
 //	logger.Info("The z component of the orientation vector: ", orientation.OZ)
 //	logger.Info("The number of degrees that the movement sensor is rotated about the vector: ", orientation.Theta)
 //
+// For more information, see the [Orientation method docs].
+//
 // Properties example:
 //
 //	// Get the supported properties of the movement sensor.
 //	properties, err := myMovementSensor.Properties(context.Background(), nil)
 //
+// For more information, see the [Properties method docs].
+//
 // Accuracy example:
 //
 //	// Get the accuracy of the movement sensor.
 //	accuracy, err := myMovementSensor.Accuracy(context.Background(), nil)
 //
-// [movement sensor component docs]: https://docs.viam.com/components/movement-sensor/
+// For more information, see the [Accuracy method docs].
+//
+// [movement sensor component docs]: https://docs.viam.com/operate/reference/components/movement-sensor/
```

Comment:
```suggestion
// [movement sensor component docs]: https://docs.viam.com/dev/reference/apis/components/movement-sensor/
```
[opt] Same as comment on the other PR--if you want, change all of these to the API pages so that we don't have to update them when we consolidate the component info onto a single page for each component. I think the overall component API page works for this for now anyway, since this is SDK reference and also the API page links to the other component page.